phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -86,6 +91,8 @@
     private static Map<String, String> serverProps = Maps.newHashMapWithExpectedSize(1),
             clientProps = Maps.newHashMapWithExpectedSize(1);
 
+    public static final String ASSERT_MESSAGE = "view-index count in system table doesn't match";
```

Comment:
nit: better constant name.

Reply:
Is this one better, VERIFY_COUNT_ASSERT_MESSAGE?